index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
+ 
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <title>mobile skeleton</title>
+</head>
+<body>
+
+    <header>
+        <nav>
+
+            <a href="#"><h2 id="welcome">Welcome</h2></a>
+            <div class="nav-links">
+            <i id="menu" class="material-symbols-outlined">menu</i>
+        </div>
+        </nav>
+        <div class="headline">
+            <h1 class="black">Hey there. I’m Jemutai.<br><span id="green">I’m a software developer</span>
+                </h1>
+                <p class="text"> 
+                    I can help you build a product , feature or website Look through some of my work and<br> experience! If you like what you see and have a project you need coded,<br>
+                   don’t hestiate to contact me.
+                </p>
+                <ul class="social-icons">
+                    <li>
+                        <a href="#"><img src="images/Vector.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Linkedinicon.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector1.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector2.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector3.png" alt="social media icons"></a>
+                    </li>
+                </ul>
```

Comment:
- [ ] Kindly add more padding/ margin left/ right of the social media icons so that it matches the chosen Figma design.

**This is how I am seeing your design:**

![image](https://user-images.githubusercontent.com/86318284/185191179-09c44f4e-cc5a-45f1-99ad-40e062072c9c.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185191294-ba837f23-5e9c-407f-b4f4-b82ce8b34a20.png)